### PR TITLE
Ima hotfix version preparation

### PR DESCRIPTION
**Pull request by payvint**

### CODE REVIEW

**Review by sergiy-skalelabs — APPROVED** [review]

File: proxy/test/MessageProxy.ts [hunk 1]
```diff
@@ -271,32 +273,34 @@ describe("MessageProxy", () => {
                 .postIncomingMessages(
                     schainName,
                     startingCounter,
-                    outgoingMessages,
+                    [message1, message1, message1, message1, message1, message1, message1, message1, message1, message1, message1],
```
Comment (yavrsky) [inline proxy/test/MessageProxy.ts]: Array(11).fill(message1)

File: proxy/test/utils/deploy/test/messageProxyCaller.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ethers } from "hardhat";
+import { MessageProxyCaller } from "../../../../typechain/MessageProxyCaller";
+
+const name = "MessageProxyCaller";
+
+export async function deployMessageProxyCaller() {
+    const factory = await ethers.getContractFactory(name);
+
+        const instance = await factory.deploy() as MessageProxyCaller;
```
Comment (yavrsky) [inline proxy/test/utils/deploy/test/messageProxyCaller.ts]: Progressive pythonic codestyle?)

File: proxy/test/MessageProxy.ts (hunk 1 above)
Comment (payvint) [reply proxy/test/MessageProxy.ts]: Hmm good notice will change it

File: proxy/test/utils/deploy/test/messageProxyCaller.ts (hunk 2 above)
Comment (payvint) [reply proxy/test/utils/deploy/test/messageProxyCaller.ts]: Yeap hybrid programming

**Review by yavrsky — APPROVED** [review]

**Review by EvgeniyZZ — APPROVED** [review]

**Review by gannakulikova — APPROVED** [review]

**Review by DimaStebaev — APPROVED** [review]

**Review by yavrsky — APPROVED** [review]

**Review by DimaStebaev — APPROVED** [review]

**Pull request merged**